wiki.py

```diff
@@ -62,6 +62,68 @@
 logger.setLevel(logging.ERROR)
 
 
+def git_repo_init() -> git.Repo:
+    """
+    Function that initializes the git repo of the Wiki (if there is one).
+    :return: initialized repo
+    """
+    try:
+        repo = git.Repo.init(WIKI_DIRECTORY)
```

Comment:
I get 
```[2022-03-14 19:58:39,236] ERROR in wiki: Error during git initialization: Cmd('git') failed due to: exit code(1)
  cmdline: git checkout master
  stderr: 'error: pathspec 'master' did not match any file(s) known to git'
```
So the initializing is not working with a fresh repo

Reply:
I took a quick look at it, but need to view more in depth to find the problem.

Reply:
I'll try to look at it later.
Thanks for the report back!

Reply:
At a first sight, instead of:
```pythhon
 repo = git.Repo.init(WIKI_DIRECTORY)
```
Should be:
```python
repo = git.Repo.init(CONFIG["wiki_directory"])
```

But I don't think that fixes the problem (I can't test it now).